src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -124,21 +130,34 @@ def get_responses(self) -> pd.DataFrame:
             "sex",
             "race_ethnicity",
             "census_year",
+            "guardian_1",
+            "guardian_1_address",
+            "guardian_2",
+            "guardian_2_address",
+            "housing_type",
         ])
 
     def to_observe(self, event: Event) -> bool:
-        if (self.clock().year % 10 == 0) & (self.clock().month == 4):
-            if self.clock().day < 29:  # because we only want one observation in April  FIXME: cooler to do this with the timestep
-                return True
-        return False
-
+        return ((self.clock().year % 10 == 0)  # decennial year
```

Comment:
shouldn't this be event time (not clock time)? The observer is doing its thing on collect_metrics.